src/main/kotlin/io/github/gradlenexus/publishplugin/TransitionCheckOptions.kt

```diff
@@ -20,9 +20,10 @@ import org.gradle.api.model.ObjectFactory
 import org.gradle.kotlin.dsl.property
 import java.time.Duration
 
-open class RetryingConfig constructor(objects: ObjectFactory) {
+@Suppress("UnstableApiUsage")
+open class TransitionCheckOptions(objects: ObjectFactory) {
 
     val maxRetries = objects.property<Int>().value(60)
 
-    val delayBetween = objects.property<Duration>().value(Duration.ofSeconds(5))
+    val delayBetween = objects.property<Duration>().value(Duration.ofSeconds(10))
```

Comment:
@szpak Shouldn't we rather increase the number of retries?

Reply:
I was thinking about that. However, a call to Sonatype Nexus every 5 seconds from all those people releasing using our plugin could potentially have a negative impact on the server.

Reply:
Nevertheless, probably I exaggerate and both cases are ok (with a few seconds shorter information about successful release in the other case).

Reply:
Fair point, I guess waiting a few seconds longer is acceptable given the other timeouts etc. here.